scripts/run_acceptance_tests.py

```diff
@@ -0,0 +1,299 @@
+                'PORTSERVER_ADDRESS': common.PORTSERVER_SOCKET_FILEPATH,
+            }))
+
+        # if (args.mobile) and (args.suite not in MOBILE_SUITES):
+        #     print(
+        #         f'The {args.suite} suite should not be run ' +
+        #         'in the mobile viewport'
+        #         )
+        #     sys.exit(1)
+
+        # proc = stack.enter_context(servers.managed_webdriverio_server(
+        #         suite_name=args.suite,
+        #         dev_mode=dev_mode,
+        #         debug_mode=args.debug_mode,
+        #         chrome_version=args.chrome_driver_version,
+        #         sharding_instances=args.sharding_instances,
+        #         mobile=args.mobile,
+        #         stdout=subprocess.PIPE))
+
+        # print(
+        #     'Servers have come up.\n'
+        #     'Note: You can view screenshots of failed tests '
+        #     'in ../webdriverio-screenshots/')
+
+        # output_lines = []
+        # while True:
+        #     # Keep reading lines until an empty string is returned. Empty
+        #     # strings signal that the process has ended.
+        #     for line in iter(proc.stdout.readline, b''):
+        #         if isinstance(line, str):
+        #             # Although our unit tests always provide unicode strings,
+        #             # the actual server needs this failsafe since it can output
+        #             # non-unicode strings.
+        #             line = line.encode('utf-8')  # pragma: no cover
+        #         output_lines.append(line.rstrip())
+        #         # Replaces non-ASCII characters with '?'.
+        #         common.write_stdout_safe(line.decode('ascii', errors='replace'))
+        #     # The poll() method returns None while the process is running,
+        #     # otherwise it returns the return code of the process (an int).
+        #     if proc.poll() is not None:
+        #         break
+
+        # return_value = output_lines, proc.returncode
+    # return return_value
+    return [], 1
+
+
+def main(args: Optional[List[str]] = None) -> None:
+    """Run tests, rerunning at most MAX_RETRY_COUNT times if they flake."""
+    parsed_args = _PARSER.parse_args(args=args)
+
+    with servers.managed_portserver():
+        for attempt_num in range(1, MAX_RETRY_COUNT + 1):
+            print('***Attempt %d.***' % attempt_num)
+            output, return_code = run_tests(parsed_args)
+
+            if not flake_checker.check_if_on_ci():
+                # Don't rerun off of CI.
+                print('No reruns because not running on CI.')
+                break
+
+            if return_code == 0:
+                # Don't rerun passing tests.
+                flake_checker.report_pass(parsed_args.suite)
+                break
+
+            # Check whether we should rerun based on the instructions from the
+            # flake checker server.
+            rerun = flake_checker.check_test_flakiness(
+                output, parsed_args.suite)
```

Comment:
Here you should create a loop that will run the "test" case. Thus it will restart the servers multiple times. In the job output this should be visible. There should be a few "starting" lines and then few "stopping" lines and this repeated.

Reply:
I am unable to figure out where to add loop and with what conditions